.circleci/config.yml

```diff
@@ -158,6 +158,7 @@ jobs:
                       - v0.5-repository_consistency
             - run: pip install --upgrade pip
             - run: pip install .[all,quality]
+            - run: pip install pytest timeout_decorator parameterized
```

Comment:
These extra 3 dependencies will allow all PyTorch model tester classes to be checked.